bin/run-yarn-audit.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+set +e
+yarn audit
+result=$?
+set -e
+
+if [[ "$result" != 0 ]]; then
+  if [[ -f yarn-audit-known-issues ]]; then
+    set +e
```

Comment:
you might have to add blank file `yarn-audit-known-issues`

Reply:
Added. Thank you.